_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+        </div>
+    </fieldset>
+    <fieldset class="field-group">
+        <legend class="field-group-hd">Additional and optional information</legend>
+        <div class="field">
+            <label class="form-label" for="gender">Gender</label>
+            <select class="form-input input-select replace-select" name="gender" id="gender">
+                <option value="">--</option>
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+                <option value="Other">Other</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="dob">Year of birth</label>
+            <select class="form-input input-select replace-select" name="dob" id="dob">
+                <option value="">--</option>
+                <option value="2015">2015</option>
+                <option value="2014">2014</option>
+                <option value="2013">2013</option>
+                <option value="2012">2012</option>
+                <option value="2011">2011</option>
+                <option value="2010">2010</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="education">Highest level of education completed</label>
+            <select class="form-input input-select replace-select" name="education" id="education">
+                <option value="">--</option>
+                <option value="Doctorate">Doctorate</option>
+                <option value="Master's or professional degree">Master's or professional degree</option>
+                <option value="Bachelor's degree">Bachelor's degree</option>
+                <option value="Associates's degree">Associates's degree</option>
+                <option value="Secondary/High School">Secondary/High School</option>
+                <option value="None">None</option>
+                <option value="Other">Other</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="mailing-address">Mailing address</label>
+            <textarea class="form-textarea input-textarea textarea-short" name="mailing-address" id="mailing-address" aria-describedby="helper-mailing-address"></textarea>
+            <div class="helper-text" id="helper-mailing-address">
+                <p>Where you'll be receiving your ACME packages.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="form-label label-inline field-required" for="agree-to-terms">
+                <input type="checkbox" class="form-input input-checkbox replace-checkbox" name="agree-to-terms" id="agree-to-terms" value="I agree to the terms and conditions." required>
+                <span class="form-input-label">I agree to the ACME <a href="#">Terms and Conditions</a></span>
+            </label>
+        </div>
+    </fieldset>
+    <div class="wrapper-form-buttons">
```

Comment:
NIT: does this class really need the prefix of ``wrapper-``? This seems like a logical spot to have a structural child of the ``.form`` element named ``.form-actions``.